src/components/admin/reference-array-input.tsx

```diff
@@ -103,5 +103,4 @@ export interface ReferenceArrayInputProps
   extends InputProps,
     UseReferenceArrayInputParams {
   children?: ReactElement;
-  label?: string;
```

Comment:
`InputProps` already defines an optional `label` prop (even though its type is wrong too, to be fixed in `ra-core`